agent-identity/python/on-behalf-of-flow/vercel-ai/main.py

```
@@ -0,0 +1,107 @@
+
+        auth_url, state, code_verifier = auth_manager.get_authorization_url_with_pkce(["openid", "email"])
+
+        print("Open this URL in your browser to authenticate:")
+        print(auth_url)
+
+        callback = OAuthCallbackServer(port=6274)
+        callback.start()
+
+        print("Waiting for authorization code from redirect...")
+
+        auth_code, returned_state, error = await callback.wait_for_code()
```

Ruff (0.15.2)

[warning] Unpacked variable `returned_state` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)